sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -167,24 +175,40 @@ export class QueueClient implements Client {
 }
 
 // @public
-export interface QueueDetails extends QueueOptions {
+export interface QueueDetails {
     accessedAt?: string;
+    authorizationRules?: AuthorizationRule[];
     createdAt?: string;
+    deadLetteringOnMessageExpiration: boolean;
+    defaultMessageTimeToLive: string;
```

Comment:
Can we change this to `defaultMessageTtl`?

Reply:
updated - looks like there was a discussion amongst team on renaming/not renaming things
does this change anything with what we decided to go with on naming of `autoDeleteOnIdle`?